### ISSUE TITLE
Really great to see work continue on this project!!

### ISSUE THREAD

**Issue by rheos** [brunoblock/oystermesh#18]
Happy to see the progress.   I wonder if at some point you will join again with a marketing team to get people using it or some other dissemination strategy?

**Comment by funkydrummer** [comment]
Dude, this guy is a scammer! lmao

https://www.justice.gov/usao-sdny/pr/cryptocurrency-founder-bruno-block-charged-multimillion-dollar-tax-evasion-scheme

**Comment by kumar987** [comment]
> Dude, this guy is a scammer! lmao
> 
> https://www.justice.gov/usao-sdny/pr/cryptocurrency-founder-bruno-block-charged-multimillion-dollar-tax-evasion-scheme

haha banks are the ones who scams, this guy is a genius.

**Comment by rheos** [comment]
yes, i heard he got charged..  too bad..  i would have loved to see where to software would have gone.

**Comment by rheos** [comment]
i mean have you even looked at the code here?   i don't know where you are at but to me this is some pretty high level stuff.

**Comment by Stemajo1908** [comment]
I can understand both positive and negative reactions.

This guy is not charged for stealing a toothbrush at the supermarket... He took advantage of his position within the company, ruined (big) investments for a lot of people, backstabbed his colleagues and took off with loads of money.
I can imagine most people who were involved in Oyster Pearl ‘ in any way’ will be satisfied to see he now gets his punnishment.

On the other hand... With a lot of interest I was following his progress on this new project too.

**Comment by kumar987** [comment]
oyster is gold

**Comment by rheos** [comment]
I think it's obvious Bruno is very smart and Oyster could be something really cool.   

But Bruno should have not stolen money from good people.   I wonder if there is any update on the trial.

**Comment by Stemajo1908** [comment]
“ Dude, this guy is a scammer! lmao “
> 
> 
And how should we call you @funkydrummer ?
